### PR TITLE
Fix deprecation warning for pytest 3.0 - use of getfuncargvalue is deprecated, use getfixturevalue

### PR DESCRIPTION
**Pull request by joshcrim**
Replace getfuncargvalue with getfixturevalue

---
This change is [Reviewable](https://reviewable.io/reviews/pytest-dev/pytest-factoryboy/49)

### CODE REVIEW

**Comment by joshcrim** [comment]
Ah, sorry, this is my second PR on github and first time using tox. Did not run tox locally before submitting the PR. Will look over this some more.

File: pytest_factoryboy/compat.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"""pytest-factoryboy compatibility."""
+
+
+def get_fixture_value(request, *args):
+    """Compatibility check for pytest2 FixtureRequest.getfunctargvalue method."""
```
Comment (blueyed) [inline pytest_factoryboy/compat.py]: s/getfunctargvalue/getfixturevalue/ ? (or remove the 2nd `t`)

**Comment by blueyed** [comment]
`getfixturevalue` was added in 2.7.3-1128-gdf9918ed (i.e. 2.8.0).
I'm not sure we should provide backward compatibility for older pytest versions?!

**Comment by joshcrim** [comment]
Are you sure? The pytest CHANGELOG says it was added in 3.0, and the tox tests for versions <3.0 fail with an AttributeError for getfixturevalue()

**Comment by blueyed** [comment]
No, I'm not sure, just assumed it from the git-describe.
It's likely that it was only merged after/for 3.0 indeed.

**Comment by joshcrim** [comment]
Ah ok, cool, I haven't done a ton of digging into pytest and thought I'd missed something there.

**Comment by olegpidsadnyi** [comment]
Let's create a major release targeting latest pytest instead (reason: also incompatible with factory_boy). I'll work on that.

**Pull request closed without merging**